OsmAnd/res/values/strings.xml

```diff
@@ -14,26 +14,26 @@
     <string name="gps_filter_precision">GPS precision</string>
     <string name="shared_string_precision">Precision</string>
     <string name="shared_string_statistics">Statistics</string>
-    <string name="gps_filter_hdop_desc">Set the maximum accepted value for HDOP.\nPoints with higher value will be hidden.</string>
-    <string name="gps_filter_speed_altitude_desc">Only track points matching the set interval will appear on the chart and map, the rest will be hidden.</string>
-    <string name="gps_filter_smoothing_desc">Set the threshold distance between points.\nTrack points not at least this distance away from the last visible point will be hidden.\nBe aware that high thresholds may oversimplify the track geometry.</string>
+    <string name="gps_filter_hdop_desc">Set the max. value for HDOP.\nPoints farther away are hidden.</string>
+    <string name="gps_filter_speed_altitude_desc">Only track points within the limit are visible on the chart and map.</string>
+    <string name="gps_filter_smoothing_desc">>Set the threshold of points to remove .\nBy doing so at at this distance to the last visible point and beyond, the track can be made too simplistic.</string>
     <string name="distance_between_points">Distance between points</string>
     <string name="gps_filter_smoothing">Smoothing</string>
     <string name="save_changes_into_file">Save changes to file</string>
     <string name="save_as_copy">Save as copy</string>
-    <string name="gps_filter_actions_description">OsmAnd will apply the changes to the track without updating the file. You can save changes manually.</string>
+    <string name="gps_filter_actions_description">OsmAnd only changes tracks without saving, but you can do so manually.</string>
```

Comment:
I like the original. "OsmAnd only changes tracks without saving," is a generic statement, and as such false. "OsmAnd will apply" is the correct description of this particular situation.